install/docker/worker-ls/docker-compose.yml

```diff
@@ -50,7 +48,6 @@ volumes:
   logs-volume:
 
 networks:
-  frontend:
-    driver: bridge         
-  backend:
-    driver: bridge
+  default:
+    external:
```

Comment:
One issue with setting network as `external` 

```
(base) MT-304569:console-db-es-ls-kibana haile$ docker-compose up
ERROR: Network cws-network declared as external, but could not be found. Please create the network manually using `docker network create cws-network` and try again.
(base) MT-304569:console-db-es-ls-kibana haile$ ls
README.md			cws-logstash.conf		worker-config.properties
config.properties		docker-compose.yml


```

It seems to work like normal with `frontend` & `backend`

Any reason for change?

Reply:
`docker network create cws-network` command should be added to the install/docker/console-db-es-ls-kibana/README.md


https://github.com/NASA-AMMOS/common-workflow-service/blob/docker-improvements/install/docker/console-db-es-ls-kibana/README.md#:~:text=To%20run%20use%20the%20command